src/vivarium/framework/results/manager.py

```diff
@@ -179,6 +179,12 @@ def on_report(self, event: Event) -> None:
                     )
 
     def gather_results(self, event_name: str, event: Event) -> None:
+        """Update the existing metrics with new results. Typically, a results
```

Comment:
I don't think we need to say what is typical and atypical behavior. Both 1 column or multiple columns are accepted inputs, so we can just say that we support each and how.